### DISCUSSION TITLE
Coreminer Proxy

### DISCUSSION THREAD

**Discussion by Tohelo101** [catchthatrabbit/coreminer#81] [Wishlist] [answered]
Hello
Is there avaible proxy for coreminer ? looking like https://github.com/xmrig/xmrig-proxy this kind thing ?

**Comment by rastislavcore** [comment]
API is documented here: https://github.com/catchthatrabbit/coreminer/blob/master/docs/API.md

**Reply by Tohelo101** [reply]
Yeah i did see that but like that xmrig-proxy makes only one connection on pool. is there something similar like that avaible on coreminer

**Reply by rastislavcore** [reply, marked as answer]
Probably you can adjust some established software. But nothing official from CTR yet. We may consider add it to TODO list.

**Reply by rastislavcore** [reply]
This is now added for the readmap. But no ETA yet. Please, if this feature is esential, let us know.

**Reply by Tohelo101** [reply]
I think it would be realy nice additional to mining. Not sure would be mining performance any better,but would think overall would better that just single device is connecting pool than 300 from one miner
